test/testfixture/make_functions.go

```diff
@@ -487,3 +489,27 @@ func makeLabels(fxt *TestFixture) error {
 	}
 	return nil
 }
+
+func makeTrackers(fxt *TestFixture) error {
+	if fxt.info[kindTrackers] == nil {
+		return nil
+	}
+	fxt.Trackers = make([]*app.Tracker, fxt.info[kindTrackers].numInstances)
+	trackerRepo := remoteworkitem.NewTrackerRepository(fxt.db)
+
+	for i := range fxt.Trackers {
+		fxt.Trackers[i] = &app.Tracker{
```

Comment:
I didn't notice this at first, but why do you create an `app.Tracker` and not a `remoteworkitem.Tracker`?

Reply:
because tracker create function returns `app.Tracker` unlike other components.

Reply:
This is reaaaaaally ugly.

Reply:
@aslakknutsen do you think we should fix this so that the `remoteworkitem.GormTrackerRepository.Create(...)` returns a `remoteworkitem.Tracker` item instead of `app.Tracker`?

Reply:
@kwk can we merge this pr and then work on the fix?

Reply:
> @kwk can we merge this pr and then work on the fix?

yes, will you do it?

Reply:
yes

Reply:
@kwk the fix discussed above can be found here -> https://github.com/fabric8-services/fabric8-wit/pull/1722